CONTRIBUTING.md

```diff
@@ -17,7 +19,6 @@ Glad you're here and interested in expanding this project 🎉 In order to make
   - [Designing Pixel Perfect Icons](#pixel-perfect-icons)
   - [Cloning existing icons](#icon-cloning)
 - [Add translations](#add-translations)
-- [Update API](#update-api)
```

Comment:
Could you please replace this point in the table of contents instead of removing it? After the "add translations" section there's another "Debug extension locally" section.

Reply:
Done